src/MadeTech/CacheWarming/Gateway/FileGetContentsUrlRetriever.php

```diff
@@ -7,6 +7,6 @@ class FileGetContentsUrlRetriever implements UrlRetriever
 {
     public function get($url)
     {
-        return file_get_contents($url);
+        return @file_get_contents($url);
```

Comment:
Spun this off into a separate issue https://github.com/madetech/cache-warming/issues/10 I'm not sure it's important yet